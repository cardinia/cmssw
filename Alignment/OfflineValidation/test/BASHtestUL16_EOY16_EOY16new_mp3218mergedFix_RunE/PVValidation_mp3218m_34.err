27-Sep-2019 21:43:07 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/76C99998-E607-E911-83AD-1866DA7F93A3.root
27-Sep-2019 21:43:10 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/76C99998-E607-E911-83AD-1866DA7F93A3.root
Begin processing the 1st record. Run 277194, Event 3469153836, LumiSection 1961 on stream 0 at 27-Sep-2019 21:43:17.169 CEST
Begin processing the 1001st record. Run 277194, Event 3475774330, LumiSection 1965 on stream 0 at 27-Sep-2019 21:47:24.797 CEST
Begin processing the 2001st record. Run 277194, Event 3484075992, LumiSection 1970 on stream 0 at 27-Sep-2019 21:51:09.204 CEST
Begin processing the 3001st record. Run 277194, Event 3492403812, LumiSection 1975 on stream 0 at 27-Sep-2019 21:54:58.850 CEST
Begin processing the 4001st record. Run 277194, Event 3500971988, LumiSection 1980 on stream 0 at 27-Sep-2019 21:58:34.276 CEST
Begin processing the 5001st record. Run 277194, Event 3509134839, LumiSection 1985 on stream 0 at 27-Sep-2019 22:01:48.407 CEST
Begin processing the 6001st record. Run 277194, Event 3516484204, LumiSection 1989 on stream 0 at 27-Sep-2019 22:05:09.306 CEST
Begin processing the 7001st record. Run 277194, Event 3523359382, LumiSection 1994 on stream 0 at 27-Sep-2019 22:08:18.938 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:10:27 CEST  Run: 277194 Event: 3530567220
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 3
%MSG
Begin processing the 8001st record. Run 277194, Event 3531834041, LumiSection 1999 on stream 0 at 27-Sep-2019 22:11:25.605 CEST
Begin processing the 9001st record. Run 277194, Event 3540661702, LumiSection 2004 on stream 0 at 27-Sep-2019 22:14:39.071 CEST
Begin processing the 10001st record. Run 277194, Event 3548713675, LumiSection 2009 on stream 0 at 27-Sep-2019 22:18:01.111 CEST
Begin processing the 11001st record. Run 277194, Event 3554260016, LumiSection 2013 on stream 0 at 27-Sep-2019 22:21:36.212 CEST
Begin processing the 12001st record. Run 277194, Event 3563499878, LumiSection 2018 on stream 0 at 27-Sep-2019 22:25:12.810 CEST
Begin processing the 13001st record. Run 277194, Event 3570769252, LumiSection 2023 on stream 0 at 27-Sep-2019 22:29:04.959 CEST
Begin processing the 14001st record. Run 277194, Event 3579925175, LumiSection 2028 on stream 0 at 27-Sep-2019 22:32:53.481 CEST
Begin processing the 15001st record. Run 277194, Event 3587178661, LumiSection 2032 on stream 0 at 27-Sep-2019 22:36:13.046 CEST
Begin processing the 16001st record. Run 277194, Event 3595953138, LumiSection 2036 on stream 0 at 27-Sep-2019 22:38:49.635 CEST
Begin processing the 17001st record. Run 277194, Event 3603337755, LumiSection 2040 on stream 0 at 27-Sep-2019 22:41:11.775 CEST
Begin processing the 18001st record. Run 277194, Event 3612428943, LumiSection 2045 on stream 0 at 27-Sep-2019 22:43:36.496 CEST
Begin processing the 19001st record. Run 277194, Event 3621334937, LumiSection 2049 on stream 0 at 27-Sep-2019 22:46:09.152 CEST
Begin processing the 20001st record. Run 277194, Event 3629551153, LumiSection 2053 on stream 0 at 27-Sep-2019 22:48:38.109 CEST
Begin processing the 21001st record. Run 277194, Event 3636230539, LumiSection 2057 on stream 0 at 27-Sep-2019 22:51:46.148 CEST
Begin processing the 22001st record. Run 277194, Event 3644340989, LumiSection 2061 on stream 0 at 27-Sep-2019 22:54:59.504 CEST
Begin processing the 23001st record. Run 277194, Event 3651301020, LumiSection 2065 on stream 0 at 27-Sep-2019 22:58:18.922 CEST
27-Sep-2019 23:00:23 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/76C99998-E607-E911-83AD-1866DA7F93A3.root
27-Sep-2019 23:00:23 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/EC3C6E51-CA07-E911-B953-842B2B6AE857.root
27-Sep-2019 23:00:25 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/EC3C6E51-CA07-E911-B953-842B2B6AE857.root
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:01:23 CEST  Run: 277194 Event: 1570573731
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1
%MSG
Begin processing the 24001st record. Run 277194, Event 1572022190, LumiSection 907 on stream 0 at 27-Sep-2019 23:01:39.080 CEST
Begin processing the 25001st record. Run 277194, Event 1579059366, LumiSection 911 on stream 0 at 27-Sep-2019 23:04:39.904 CEST
Begin processing the 26001st record. Run 277194, Event 1586793542, LumiSection 915 on stream 0 at 27-Sep-2019 23:07:35.442 CEST
Begin processing the 27001st record. Run 277194, Event 1594924821, LumiSection 920 on stream 0 at 27-Sep-2019 23:11:06.251 CEST
Begin processing the 28001st record. Run 277194, Event 1603481350, LumiSection 924 on stream 0 at 27-Sep-2019 23:15:03.982 CEST
Begin processing the 29001st record. Run 277194, Event 1610237060, LumiSection 928 on stream 0 at 27-Sep-2019 23:19:16.418 CEST
Begin processing the 30001st record. Run 277194, Event 1618859402, LumiSection 932 on stream 0 at 27-Sep-2019 23:23:25.311 CEST
Begin processing the 31001st record. Run 277194, Event 1626752133, LumiSection 936 on stream 0 at 27-Sep-2019 23:27:41.825 CEST
Begin processing the 32001st record. Run 277194, Event 1636899312, LumiSection 941 on stream 0 at 27-Sep-2019 23:31:55.579 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:36:11 CEST  Run: 277194 Event: 1644427395
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 3610
%MSG
Begin processing the 33001st record. Run 277194, Event 1644931686, LumiSection 945 on stream 0 at 27-Sep-2019 23:36:11.938 CEST
Begin processing the 34001st record. Run 277194, Event 1652533482, LumiSection 949 on stream 0 at 27-Sep-2019 23:40:21.945 CEST
Begin processing the 35001st record. Run 277194, Event 1658772411, LumiSection 953 on stream 0 at 27-Sep-2019 23:44:39.368 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:45:21 CEST  Run: 277194 Event: 1661488252
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 36001st record. Run 277194, Event 1669474434, LumiSection 958 on stream 0 at 27-Sep-2019 23:48:44.814 CEST
Begin processing the 37001st record. Run 277194, Event 1677790947, LumiSection 962 on stream 0 at 27-Sep-2019 23:52:58.928 CEST
Begin processing the 38001st record. Run 277194, Event 1685635619, LumiSection 966 on stream 0 at 27-Sep-2019 23:57:00.143 CEST
Begin processing the 39001st record. Run 277194, Event 1692321649, LumiSection 970 on stream 0 at 28-Sep-2019 00:01:09.367 CEST
Begin processing the 40001st record. Run 277194, Event 1700987165, LumiSection 974 on stream 0 at 28-Sep-2019 00:05:11.259 CEST
Begin processing the 41001st record. Run 277194, Event 1709390843, LumiSection 979 on stream 0 at 28-Sep-2019 00:09:23.023 CEST
Begin processing the 42001st record. Run 277194, Event 1718170036, LumiSection 983 on stream 0 at 28-Sep-2019 00:13:31.687 CEST
Begin processing the 43001st record. Run 277194, Event 1724581625, LumiSection 987 on stream 0 at 28-Sep-2019 00:17:32.286 CEST
Begin processing the 44001st record. Run 277194, Event 1732221004, LumiSection 991 on stream 0 at 28-Sep-2019 00:21:18.448 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 28-Sep-2019 00:22:27 CEST  Run: 277194 Event: 1737300132
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 3
%MSG
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 28-Sep-2019 00:23:09 CEST  Run: 277194 Event: 1735941446
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 45001st record. Run 277194, Event 1740156982, LumiSection 995 on stream 0 at 28-Sep-2019 00:25:00.837 CEST
Begin processing the 46001st record. Run 277194, Event 1747278651, LumiSection 999 on stream 0 at 28-Sep-2019 00:28:51.866 CEST
Begin processing the 47001st record. Run 277194, Event 1757860329, LumiSection 1004 on stream 0 at 28-Sep-2019 00:32:58.831 CEST
Begin processing the 48001st record. Run 277194, Event 1764767027, LumiSection 1008 on stream 0 at 28-Sep-2019 00:36:46.274 CEST
Begin processing the 49001st record. Run 277194, Event 1772081926, LumiSection 1012 on stream 0 at 28-Sep-2019 00:39:59.927 CEST
28-Sep-2019 00:42:48 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016E/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/10000/EC3C6E51-CA07-E911-B953-842B2B6AE857.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       6        6
    2 fileAction           -s file_close                             2        2
    3 fileAction           -s file_open                              4        4

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          Run: 277194 Even 277194/157057373 277194/173594144
    2 fileAction           PostProcessEvent PostGlobalEndRun 
    3 fileAction           pre-events       pre-events       PostProcessEvent

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 6                   6
System                  6                   6

dropped waiting message count 0
[9.798MB/9.798MB][100%][==================================================][9.798MB/s]  [9.798MB/9.798MB][100%][==================================================][9.798MB/s]  
