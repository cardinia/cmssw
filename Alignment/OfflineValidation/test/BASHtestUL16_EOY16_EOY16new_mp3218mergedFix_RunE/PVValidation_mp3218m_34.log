000 (3196552.034.000) 09/27 21:32:53 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196552.034.000) 09/27 21:42:57 Started transferring input files
	Transferring to host: <188.184.183.163:9618?addrs=188.184.183.163-9618+[2001-1458-301-2b--100-176]-9618&noUDP&sock=5126_7b0a_13052>
...
040 (3196552.034.000) 09/27 21:42:57 Finished transferring input files
...
001 (3196552.034.000) 09/27 21:42:57 Job executing on host: <188.184.183.163:9618?addrs=188.184.183.163-9618+[2001-1458-301-2b--100-176]-9618&noUDP&sock=3312_55c2_3>
...
006 (3196552.034.000) 09/27 21:43:06 Image size of job updated: 235872
	119  -  MemoryUsage of job (MB)
	121788  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 21:48:07 Image size of job updated: 661760
	535  -  MemoryUsage of job (MB)
	547548  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 21:53:08 Image size of job updated: 662816
	536  -  MemoryUsage of job (MB)
	548604  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 22:03:09 Image size of job updated: 662856
	536  -  MemoryUsage of job (MB)
	548640  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 22:08:10 Image size of job updated: 663100
	536  -  MemoryUsage of job (MB)
	548860  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 22:18:10 Image size of job updated: 663356
	537  -  MemoryUsage of job (MB)
	549116  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 22:23:11 Image size of job updated: 664540
	538  -  MemoryUsage of job (MB)
	550300  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 22:38:12 Image size of job updated: 666188
	540  -  MemoryUsage of job (MB)
	551948  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 23:03:15 Image size of job updated: 688400
	561  -  MemoryUsage of job (MB)
	574120  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 23:13:15 Image size of job updated: 689552
	562  -  MemoryUsage of job (MB)
	575272  -  ResidentSetSize of job (KB)
...
006 (3196552.034.000) 09/27 23:58:20 Image size of job updated: 690812
	564  -  MemoryUsage of job (MB)
	576532  -  ResidentSetSize of job (KB)
...
040 (3196552.034.000) 09/28 00:43:08 Started transferring output files
...
040 (3196552.034.000) 09/28 00:43:08 Finished transferring output files
...
005 (3196552.034.000) 09/28 00:43:08 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 02:56:07, Sys 0 00:00:19  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 02:56:07, Sys 0 00:00:19  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	11146  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	11146  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :    10084        1   2880198 
	   Memory (MB)          :      564     2000      2000 
...
