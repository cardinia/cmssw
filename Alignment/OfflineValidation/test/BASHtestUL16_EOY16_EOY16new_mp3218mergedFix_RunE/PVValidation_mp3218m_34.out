Job started at  Fri Sep 27 21:42:57 CEST 2019
batch dir: /pool/condor/dir_18151 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_277194_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.9M -rw-r--r--. 1 tkello zh 9.8M Sep 28 00:43 PVValidation_mp3218m_277194.root -rw-r--r--. 1 tkello zh 22K Sep 27 21:42 PVValidation_mp3218m_277194_cfg.py -rw-r--r--. 1 tkello zh 9.7K Sep 28 00:43 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:43 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:42 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:43 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:42 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:42 var
Job ended at  Sat Sep 28 00:43:08 CEST 2019
